drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -323,6 +333,7 @@ local function do_configure(driver, device)
   local thermo_eps = device:get_endpoints(clusters.Thermostat.ID)
   local fan_eps = device:get_endpoints(clusters.FanControl.ID)
   local wind_eps = device:get_endpoints(clusters.FanControl.ID, {feature_bitmap = clusters.FanControl.types.FanControlFeature.WIND})
```

Comment:
Does `FanControlFeature` need to be changed to `Feature`, or does aliasing take care of this?

Reply:
Aliasing will take care of this. However, I don't think it's the worst idea to go ahead and update all the places where this old feature name is used. Thermostat and Switch also use this old naming convention.